packages/kbn-optimizer/limits.yml

```diff
@@ -91,7 +91,7 @@ pageLoadAssetSize:
   licensing: 29004
   links: 44490
   lists: 22900
-  logExplorer: 54342
+  logExplorer: 102152
```

Comment:
Are we able to load this asynchronously?

Reply:
I will check this.

Reply:
@jbudz Thank you for the suggestion. I changed the logic to lazy load dependent components and now the bundle size is smaller